libraries/ms-common/src/stm32f0xx/gpio.c

```diff
@@ -35,11 +35,6 @@ static bool prv_are_settings_valid(const GPIOSettings *settings) {
 
 
 StatusCode gpio_init(void) {
```

Comment:
Probably not. That would require knowledge of UART retargeting within ms-common, and I'd prefer for this to be part of the STM library.